### PR TITLE
Changed Kusto Description

### PR DESCRIPTION
**Pull request by oflipman**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Kusto/Management.Kusto/Microsoft.Azure.Management.Kusto.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Kusto</PackageId>
-    <Description>Provides developers with libraries for the updated kusto platform under Azure Resource manager to create ane manage clusters. Note: This client library is for Virtual Machines under Azure Resource Manager.</Description>
```
Comment (dsgouda) [inline src/SDKs/Kusto/Management.Kusto/Microsoft.Azure.Management.Kusto.csproj]: Description cannot be changed without bumping the version number here. What's the intent of this change?

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/Kusto/Management.Kusto/Microsoft.Azure.Management.Kusto.csproj (hunk 1 above)
Comment (oflipman) [reply src/SDKs/Kusto/Management.Kusto/Microsoft.Azure.Management.Kusto.csproj]: The current description is wrong.

File: src/SDKs/Kusto/Management.Kusto/Microsoft.Azure.Management.Kusto.csproj [hunk 2]
```diff
@@ -5,8 +5,8 @@
...
-    <Version>0.9.0-preview</Version>
+    <Description>Provides management capabilities for Kusto (Azure Data Explorer) clusters.</Description>
```
Comment (dsgouda) [inline src/SDKs/Kusto/Management.Kusto/Microsoft.Azure.Management.Kusto.csproj]: Please remove the `Descriptions` section and move it to `PackageReleaseNotes` instead.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good apart from a minor change

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**